gnucash/gnome/assistant-stock-transaction.cpp

```diff
@@ -0,0 +1,1151 @@
+    {
+        create_split (trans, info->txn_type.capgains_value, "Capgains",
+                      gas_account (info->capgains_account),
+                      account_commits, info->capgains_memo_edit,
+                      info->capgains_value, info->capgains_value, false);
+
+        create_split (trans, info->txn_type.capgains_value ^ 0b11, "Capgains",
+                      info->acct, account_commits, info->capgains_memo_edit,
+                      nullptr, info->capgains_value, false);
+    }
+
+    // // amount = gnc_amount_edit_get_amount (GNC_AMOUNT_EDIT (info->price_edit));
+    // amount = gnc_numeric_zero (); // fixme -- price = value/amount
+    // if (gnc_numeric_positive_p (amount))
+    // {
+    //     // auto ce = GNC_CURRENCY_EDIT (info->price_currency_edit);
+    //     auto price = gnc_price_create (gnc_get_current_book ());
+
+    //     gnc_price_begin_edit (price);
+    //     gnc_price_set_commodity (price, xaccAccountGetCommodity (account));
+    //     // gnc_price_set_currency (price, gnc_currency_edit_get_currency (ce));
+    //     gnc_price_set_time64 (price, date);
+    //     gnc_price_set_source (price, PRICE_SOURCE_STOCK_SPLIT);
+    //     gnc_price_set_typestr (price, PRICE_TYPE_UNK);
+    //     gnc_price_set_value (price, amount);
+    //     gnc_price_commit_edit (price);
+
+    //     auto book = gnc_get_current_book ();
+    //     auto pdb = gnc_pricedb_get_db (book);
+
+    //     if (!gnc_pricedb_add_price (pdb, price))
+    //         gnc_error_dialog (GTK_WINDOW (info->window), "%s", _("Error adding price."));
+    // }
+
+    // amount = gnc_amount_edit_get_amount (GNC_AMOUNT_EDIT (info->cash_value));
+    // if (gnc_numeric_positive_p (amount))
+    // {
+    //     const char *memo = gtk_entry_get_text (GTK_ENTRY (info->cash_memo_edit));
+
+    //     /* asset split */
+    //     // account = gnc_tree_view_account_get_selected_account
+    //     //     (GNC_TREE_VIEW_ACCOUNT(info->asset_tree));
+    //     account_commits.emplace_back (account);
+
+    //     split = xaccMallocSplit (gnc_get_current_book ());
+    //     xaccAccountBeginEdit (account);
+    //     xaccSplitSetAccount (split, account);
+    //     xaccSplitSetParent (split, trans);
+    //     xaccSplitSetAmount (split, amount);
+    //     xaccSplitSetValue (split, amount);
+    //     xaccSplitSetMemo (split, memo);
+
+    //     /* income split */
+    //     // account = gnc_tree_view_account_get_selected_account
+    //     //     (GNC_TREE_VIEW_ACCOUNT(info->income_tree));
+    //     account_commits.emplace_back (account);
+
+    //     split = xaccMallocSplit (gnc_get_current_book ());
+    //     xaccAccountBeginEdit (account);
+    //     xaccSplitSetAccount (split, account);
+    //     xaccSplitSetParent (split, trans);
+    //     xaccSplitSetAmount (split, gnc_numeric_neg (amount));
+    //     xaccSplitSetValue (split, gnc_numeric_neg (amount));
+    //     xaccSplitSetMemo (split, memo);
+    // }
+
```

Comment:
Remove commented-out code.

Reply:
this will be fixed when the whole branch is condensed into 1 commit.